modules/MsgReader.jsm.js

```diff
@@ -98,8 +98,7 @@ var MsgReader = {
 					if (posEndHeader === -1) {
 						// in this case, the message has no body, but headers must end with a newline
 						if (!str.endsWith("\r\n")) {
-							throw new DKIM_InternalError("Message is not in correct e-mail format",
-								"DKIM_INTERNALERROR_INCORRECT_EMAIL_FORMAT");
+							throw new DKIM_Error("DKIM_INTERNALERROR_INCORRECT_EMAIL_FORMAT");
```

Comment:
`DKIM_Error` should be correct here, but it takes a general error description, not a localized string.

Reply:
The change from `DKIM_InternalError` to `DKIM_Error` was correct, and I think fits better than the `Error` it is now. This was just about the error message.